solutions/instances/variables.tf

```diff
@@ -43,6 +43,151 @@ variable "prefix" {
+# IBM Cloud Logs
+##############################################################################
+variable "cloud_logs_provision" {
+  description = "Set it to true to provision an IBM Cloud Logs instance"
+  type        = bool
+  default     = false
+}
+
+variable "cloud_logs_plan" {
+  type        = string
+  description = "The IBM Cloud Logs plan to provision. Possible values: standard"
+  default     = "standard"
+}
+
+variable "cloud_logs_instance_name" {
+  type        = string
+  description = "The name of the IBM Cloud Logs instance to create. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
+  default     = "cloud-logs"
+}
+
+variable "cloud_logs_region" {
+  description = "The region where Cloud Logs instances resources are created."
+  type        = string
+  default     = null
+}
+
+variable "cloud_logs_tags" {
+  type        = list(string)
+  description = "The resource tags that are associated with the IBM Cloud Logs instance (`Optional`, `array of strings`)."
+  default     = []
+}
+
+variable "cloud_logs_service_endpoints" {
+  description = "The type of endpoint for the IBM Cloud Logs instance. Possible values: `public`, `private`, `public-and-private`."
+  type        = string
+  default     = "public-and-private"
+  validation {
+    condition     = contains(["public", "private", "public-and-private"], var.cloud_logs_service_endpoints)
+    error_message = "The specified service_endpoints is not a valid selection"
+  }
+}
+
+variable "enable_cloud_logs_data" {
+  type        = bool
+  description = "Whether to enable cloud logs data to the Object Storage bucket."
+  default     = true
+}
+
+variable "enable_cloud_logs_metrics" {
+  type        = bool
+  description = "Whether to enable cloud logs metric data to the Object Storage bucket."
+  default     = true
+}
+
+variable "enable_en_cloud_logs_integration" {
+  type        = bool
+  description = "Whether to enable event notification integration for IBM Cloud Logs instance"
+  default     = false
+}
+
+variable "existing_en_instance_crn" {
+  type        = string
+  description = "The CRN of existing event notification instance."
+  default     = null
+}
+
+variable "cloud_logs_retention_period" {
+  type        = number
+  description = "The number of days IBM Cloud Logs will retain the logs data in priority insights."
+  default     = 7
+}
+
+variable "cloud_log_data_bucket_name" {
+  type        = string
+  default     = "cloud-logs-data-bucket"
+  description = "The name of the Cloud Object Storage bucket to create to store cloud log data. Cloud Object Storage bucket names are globally unique. If the `add_bucket_name_suffix` variable is set to `true`, 4 random characters are added to this name to ensure that the name of the bucket is globally unique. If the prefix input variable is passed, the name of the instance is prefixed to the value in the `<prefix>-value` format."
```

Comment:
`the name of the instance is prefixed ` -> `the name of the bucket is prefixed `